packages/plugin-connector/src/index.js

```diff
@@ -0,0 +1,3 @@
+export * from './build-plugins/preprocess';
+export * from './build-plugins/rollup';
+export * from './component-resolution/get-plugin-components';
```

Comment:
This doesn't need to be exported from the package.

Might be worth adding a `./internals` exports in package.json to make it accessible though.